packages/integrations/node/src/standalone.ts

```diff
@@ -9,11 +9,17 @@ import { createAppHandler } from './serve-app.js';
 import { createStaticHandler } from './serve-static.js';
 import type { Options } from './types.js';
 
+// Used to get Host Value at Runtime
+export const hostOptions = (host: string | boolean): string => {
```

Comment:
Perhaps we can use the type from `Options`

```suggestion
export const hostOptions = (host: Options["host"]): string => {
```

Reply:
Good suggestion! LGTM!